openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -456,7 +441,7 @@ def post(self, request):
                         default=settings.LANGUAGE_CODE
                     )
                     notification = DeletionNotificationMessage().personalize(
-                        recipient=Recipient(lms_user_id=0, email_address=user_email),
+                        recipient=Recipient(username='', email_address=request.user.email),
```

Comment:
Perhaps the wrong email address is used here? `request.user.email` should be the retired (invalid) email address and `user_email` is the previosuly-saved, valid, pre-retirement email address.

Reply:
You're right. Sorry about that. I overlooked it during the refactoring. Fixed it now.